timemachine/datasets/water_exchange/bb_centered_espaloma.sdf

```diff
@@ -0,0 +1,258 @@
+buckyball_3d_ammonia
```

Comment:
nit: stale metadata (doesn't contain ammonia, might remove `s_m_Source_Path` as well)

Reply:
addressed in https://github.com/proteneer/timemachine/pull/1137/commits/7a816faf942308d3ba456aa0642e25974b1d4c7e